lib/arjdbc/abstract/core.rb

```diff
@@ -74,3 +74,18 @@ def log(sql, name = "SQL", binds = [], type_casted_binds = [], statement_name =
     end
   end
 end
+
+
+module ActiveRecord
+  class LogSubscriber
+
+    JDBC_GEM_ROOT = File.expand_path("../../../..", __FILE__) + "/"
+
+    # Remove this gem from log trace so that query shows where it was called in application
+    def ignored_callstack(path)
```

Comment:
I'm not sure I follow what this is doing 100%, but won't it make it harder for us to debug issues when the problem lies in the gem/rails?

Reply:
Actually the opposite. All that method does is figure out which line executed the AR call and logs it into rails sever log if `ActiveRecord::Base.verbose_query_logs`. See [source](https://github.com/rails/rails/blob/fc5dd0b85189811062c85520fd70de8389b55aeb/activerecord/lib/active_record/log_subscriber.rb#L102). It already removes AR gem from stack trace. 

Without the fix (same line every time)

```
User Load (7.0ms)  SELECT  * FROM USERS WHERE id = ? LIMIT ?  [["id", "1"], ["LIMIT", 1]]
  ↳ /home/ruslan/rails/rsov/activerecord-jdbc-adapter/lib/arjdbc/abstract/core.rb:72
```

Expected (line changes based where it was called from)
```
User Load (7.0ms)  SELECT  * FROM USERS WHERE id = ? LIMIT ?  [["id", "1"], ["LIMIT", 1]]
  ↳ app/controllers/application_controller.rb:107


Member Load (59.2ms)  SELECT * FROM MEMBERS WHERE id = ? LIMIT ?  [["id", 1], ["LIMIT", 1]]
  ↳ app/views/layouts/_details.html.erb:6
```

Reply:
nice, thanks @rsov ... think it all makes sense.
@rdubya if you do not have anything else I'm fine with merging - feel free to go forward.

Reply:
Can you update it to create a module that we prepend to the class that then calls `super`? That way we don't have to come back and update this if they ever change what they are filtering out

Reply:
@rdubya Done

Reply:
Cool. Thanks